main.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+const isNode = require("is-node");
+
+if (process.env.NODE_ENV === 'production' && !isNode) {
+	const shim = new Proxy(function () {}, {
```

Comment:
To simplify things I used the main source code of `ow-shim` here instead of using the module itself. We could do that instead, though that would probably just increase the bundle size 😅.